README.md

```diff
@@ -26,6 +26,28 @@ If you need support for other Google APIs, please check out the [Google APIs Cli
 $ composer require google/cloud
 ```
 
+## Caching Access Tokens
+
+It is possible to cache access tokens by passing a [PSR-6](http://www.php-fig.org/psr/psr-6/) caching implementation in to the desired client.
+
+The following example takes advantage of [Stash](https://github.com/tedious/Stash).
+
+```php
+<?php
+require 'vendor/autoload.php';
+
+use Google\Cloud\Storage\StorageClient;
+use Stash\Driver\FileSystem;
+use Stash\Pool;
+
+$driver = new FileSystem([]);
```

Comment:
That's a good question. I am not certain, and after seeing https://github.com/google/google-auth-library-php/pull/138 I don't feel 100% confident in using stash as the example here. I will update.